testcafe/searchField.test.ts

```diff
@@ -0,0 +1,81 @@
+import { Console } from "console";
+import { Selector } from "testcafe";
+import { Config } from "./testcafe.config";
+
+fixture`Search field test`.page`${Config.baseUrl}`
+    .beforeEach( async t => {
+        const nextButton = Selector("a").withAttribute("role", "button").child().withText("Weiter");
+        const websiteText = Selector("html").textContent;
+        const titleForIntro1 = "Willkommen bei der CovMap";
+        const titleForIntro2 = "Was ist die CovMap?";
+        const titleForIntro3 = "Was zeigt mir die CovMap an?";
+        const titleTextForPostalCode = "Für Dein regionales Risiko brauchen wir noch die Postleitzahl Deines Wohnortes";
+        
+        await t.expect(websiteText).contains(titleForIntro1).click(nextButton);
+        await t.expect(websiteText).contains(titleForIntro2).click(nextButton);
+        await t.expect(websiteText).contains(titleForIntro3).click(nextButton);
+        await t.expect(websiteText).contains(titleTextForPostalCode).pressKey("1 0 1 1 5 tab space tab tab enter");
+        await t.expect(websiteText).notContains(titleTextForPostalCode);
+    })
+    .afterEach( async t => {
+        // place holder
+    });
+
+test("find search field", async (t) => {
+    var searchField = Selector('#autocomplete').withAttribute("placeholder", "PLZ oder Landkreis")
+    await t.expect(searchField.exists).ok();
+    await t.click(searchField).typeText(searchField, "Stuttgart").pressKey("enter");
+    // wait until the map has scrolled.
+    await t.wait(1000);
+
+    // open panel for risk description
+    var riskButton = Selector('button').withAttribute("aria-label", "show more").withAttribute("aria-expanded","false")
+    await t.expect(riskButton.exists).ok();
+    const websiteText = Selector("html").textContent;
+    await t.click(riskButton).expect(websiteText).contains("Kontaktverhalten der Bevölkerung");
```

Comment:
On master there is a new file `testUtils.ts`, which contains a function `textExists` that might be better suited here.

(When I first implemented this text search I was not aware that it is not as stable as the one new.)

Reply:
Hier I have the problem, that the button do not contains any text and I have to find it with attributes.

Reply:
we could add properties for screen readers (which we should add either way) this way you can get the button more easily.

Reply:
It would be great, should we do that in another PR?

Reply:
The aria-label goes to that direction, correct? I think the usage of the button element with the aria-label here is good.